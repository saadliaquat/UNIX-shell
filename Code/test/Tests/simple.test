ls
touch tempfile
ls
rm -f tempfile
mkdir -p random_ahh_dir
whoami
rm -rf random_ahh_dir
head -n 10 config.json
tail -n 5 config.json
cp config.json config.json.bak
ls
rm -f config.json.bak
ls
git status