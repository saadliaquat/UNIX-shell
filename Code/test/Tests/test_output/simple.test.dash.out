Tests
config.json
config.json.log
find.log
names.txt
test.py
Tests
config.json
config.json.log
find.log
names.txt
tempfile
test.py
root
{
    "test_directory": "Tests",
    "default_tests": [
        "easy",
        "medium",
        "advanced"
    ],
    "test_files": {
        "easy": [
            "simple.test",
    "reference_shell": "dash",
    "default_timeout": 3,
    "log_to_file": false,
    "verbose": true
}
Tests
config.json
config.json.bak
config.json.log
find.log
names.txt
test.py
Tests
config.json
config.json.log
find.log
names.txt
test.py
